src/modules/rtpengine/api.h

```diff
@@ -26,18 +26,17 @@
 
 #include "../../core/parser/msg_parser.h"
 
-typedef int (*rtpengine_start_recording_f)(
-		struct sip_msg *msg, char *str1, char *str2);
-typedef int (*rtpengine_answer_f)(struct sip_msg *msg, char *str1, char *str2);
-typedef int (*rtpengine_offer_f)(struct sip_msg *msg, char *str1, char *str2);
-typedef int (*rtpengine_delete_f)(struct sip_msg *msg, char *str1, char *str2);
+typedef int (*rtpengine_start_recording_f)(struct sip_msg *msg);
+typedef int (*rtpengine_answer_f)(struct sip_msg *msg, str *str);
+typedef int (*rtpengine_offer_f)(struct sip_msg *msg, str *str);
+typedef int (*rtpengine_delete_f)(struct sip_msg *msg, str *str);
 
 typedef struct rtpengine_api
 {
 	rtpengine_start_recording_f start_recording;
 	rtpengine_answer_f answer;
 	rtpengine_offer_f offer;
-	rtpengine_delete_f delete;
+	rtpengine_delete_f _delete;
```

Comment:
Any reason for the underscore? :smile:

J/W, otherwise LGTM

Reply:
Thanks for the review. I added the underscore to delete to avoid potential conflicts with the delete keyword in C++. While this code is written in C, there's a possibility that it might be included in a C++ project in the future. Using _delete ensures compatibility and prevents any issues if that happens. If you think it's unnecessary, I'm happy to revert it, but I thought it was a safer choice to avoid potential problems.

Reply:
> Thanks for the review. I added the underscore to delete to avoid potential conflicts with the delete keyword in C++. While this code is written in C, there's a possibility that it might be included in a C++ project in the future. Using _delete ensures compatibility and prevents any issues if that happens. If you think it's unnecessary, I'm happy to revert it, but I thought it was a safer choice to avoid potential problems.

Ok, fair point, but I do think having the member name with an underscore looks a bit odd. Perhaps use some alternative like `del` or `delet` then, or even just `teardown` or something else.